### DISCUSSION TITLE
More visual customization, everywhere in the game

### DISCUSSION THREAD

**Discussion by StalkyBoi43815** [ppy/osu#27182] [Ideas] [answered]
On stable, we could change the look of the whole game with a skin. However, Lazer lacks such customization. I think we should be able to change a lot about the way the game looks. For example, we see the map's background **3 times** in game, and if we open the _now playing_ popup, it's visible **4 times**!
![Screenshot_20240215_153757](https://github.com/ppy/osu/assets/148560144/899c61a7-2bb9-4b5c-94bc-82471021d87c)
![Screenshot_20240215_153703](https://github.com/ppy/osu/assets/148560144/de911e62-bc65-4421-aaf8-3de4349a4a05)
I think more freedom for changing the look of the menus and such would be great on an **open-source** game.

**Comment by Stoppedpuma** [comment]
Duplicate of #24884.

**Reply by StalkyBoi43815** [reply]
That's skin specific. I mean everything, like even the settings menu, which you couldn't customize on stable.

**Reply by Stoppedpuma** [reply, marked as answer]
If things like the settings menu were to be customisable I'm pretty sure it still applies to that conversation. It's been brought up multiple times across various discussions that lazer will be both easier and more customisable than stable.
